src/test/java/org/opentripplanner/graph_builder/module/osm/TestIntermediatePlaces.java

```diff
@@ -26,6 +26,9 @@
 import org.opentripplanner.routing.algorithm.mapping.TripPlanMapper;
```

Comment:
We could also simply get rid of this test. It's not likely to be enabled any time soon.

Reply:
Agree. We will have to rewrite all those tests anyway. I can remove this class. What do you think @t2gran ?